File: CS/2114_Maximum_Number_of_Words_Found_in_Sentences.cs
public class Solution {
    public int MostWordsFound(string[] sentences) {
        int result = Int32.MinValue;
        
        for(int i = 0; i < sentences.Length; i++){
            int temp = 1;
            for(int j = 0; j < sentences[i].Length; j++){
                if(sentences[i][j] == ' '){
                    temp += 1;
                }
            }
            result = Math.Max(result, temp);
        }
        
        return result;
    }
}

File: Cpp/2114_Maximum_Number_of_Words_Found_in_Sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int result = 0;
        for(int i = 0; i < sentences.size(); i++){
            int temp = 1;
            for(char c : sentences[i]){
                if(c == ' '){
                    temp++;
                }
            }
            result = max(result, temp);
        }
        return result;
    }
};
